src/components/CountryDetails.jsx

```diff
@@ -0,0 +1,67 @@
+import React, {useState, useEffect} from 'react'
+import {useParams} from 'react-router-dom'
```

Comment:
and let's import Link from 'react-router-dom'

`import {useParams, Link} from 'react-router-dom'`